Web/starter_project_vanguard/components/profile/PersonalInfo.tsx

```diff
@@ -0,0 +1,120 @@
+import Image  from 'next/image'
+import React from 'react'
+
+import {User} from '@/types/user-profile'
+import profile from 'public/images/profile-photo.png'
+
+const user: User = {
+  firstName: 'Segni',
+  lastName: 'Desta',
+  email: 'segni@gmail.com',
+  img: profile,
+}
+const PersonalInfo:React.FC = () => {
+  return (
+    <div className="flex flex-col gap-4 py-6">
+      <div className="flex justify-between">
+        <div className="font-montserrat">
+          <h1 className="font-semibold pt-3 text-xl">
+            Manage Personal Information
+          </h1>
+          <p className="text-primary-text">
+            Add all required information about yourself
+          </p>
+        </div>
+        <button className="bg-primary text-white rounded px-3 h-9 my-auto mr-8">
+          Save Changes
+        </button>
+      </div>
+      <hr />
+
+      <div className="flex py-5 gap-10">
+        <label htmlFor="">
+          Name <span className="text-red-500">*</span>
+        </label>
+        <div className="flex gap-10 ml-10">
+          <input
+            className="bg-gray-50  ml-10 w-[200%]  text-gray-900 b-2 pl-3 outline-none  border border-gray-300 rounded-lg"
+            value={user.firstName}
+            type="text"
+          />
+          <input
+            className="b-2 p-2 pl-3 outline-none ml-10 border border-gray-300 bg-gray-50 rounded-lg"
+            value={user.lastName}
+            type="text"
+          />
+        </div>
+      </div>
+      <hr />
+      <div className="flex py-5 gap-10">
+        <label htmlFor="" className="mr-10">
+          Email <span className="text-red-500">*</span>
+        </label>
+        <div className="relative mb-6">
+          <div className="absolute inset-y-0 left-0 flex items-center pl-3 pointer-events-none">
+            <svg
+              aria-hidden="true"
+              className="w-5 h-5 ml-10 text-gray-500 dark:text-gray-400"
+              fill="currentColor"
+              viewBox="0 0 20 20"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path d="M2.003 5.884L10 9.882l7.997-3.998A2 2 0 0016 4H4a2 2 0 00-1.997 1.884z"></path>
+              <path d="M18 8.118l-8 4-8-4V14a2 2 0 002 2h12a2 2 0 002-2V8.118z"></path>
+            </svg>
+          </div>
+          <input
+            required
+            value={user.email}
+            type="text"
+            id="input-group-1"
+            className="bg-gray-50 border ml-10 w-[195%] border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block  pl-10 p-2.5  dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"
```

Comment:
It would be better to define the colors globally in the tailwind config file so it can be reused by others.